src/components/InstrumentControls/Camera.vue

```diff
@@ -137,6 +138,29 @@
       </b-field>
     </b-field>
 
+    <b-field
+      v-if="auto_color_options.length > 1"
+      horizontal
+      label="Auto-Color"
+    >
+      <b-field>
+        <b-select
+          v-model="auto_color_options_selection"
+          placeholder="auto-color..."
+          size="is-small"
+        >
+          <option
+            v-for="(color_opt, index) in auto_color_options"
+            :key="index"
+            :value="color_opt"
+            :selected="index === 0"
```

Comment:
I don't think this is necessary because the default selection is already defined through the v-model setting.